e2e_testing/torchscript/batchnorm.py

```diff
@@ -97,7 +97,7 @@ def __init__(self):
     @export
     @annotate_args([
         None,
-        ([2, 5, 2, 2, 3], torch.float32, True),
+        ([5, 2, 2, 3], torch.float32, True),
```

Comment:
Is it possible to keep the original test but add one specifically for TOSA?

Reply:
Added one with '4D' in the name.